sublime-package.json

```diff
@@ -0,0 +1,309 @@
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.Vocabulary": {
+                      "default": true,
+                      "description": "Suggests alternatives to bland and overused words such as 'good' and 'nice'.",
+                      "type": "boolean"
+                    },
+                    "grammarly.files.exclude": {
+                      "default": [],
+                      "items": {
+                        "type": "string"
+                      },
+                      "markdownDescription": "Configure [glob patterns](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options) for excluding files and folders.",
+                      "order": 2,
+                      "required": true,
+                      "type": "array"
+                    },
+                    "grammarly.files.include": {
+                      "default": [
+                        "**/readme.md",
+                        "**/README.md",
+                        "**/*.txt"
+                      ],
+                      "items": {
+                        "type": "string"
+                      },
+                      "markdownDescription": "Configure [glob patterns](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options) for including files and folders.",
+                      "order": 1,
+                      "required": true,
+                      "type": "array"
+                    },
+                    "grammarly.patterns": {
+                      "default": [
+                        "**/readme.md",
+                        "**/README.md",
+                        "**/*.txt"
+                      ],
+                      "description": "A glob pattern, like `*.{md,txt}` for file scheme.",
+                      "items": {
+                        "type": "string"
+                      },
+                      "markdownDeprecationMessage": "Use [Files: Include](#grammarly.files.include#)",
+                      "order": 0,
+                      "required": true,
+                      "type": "array"
+                    },
+                    "grammarly.selectors": {
+                      "default": [],
+                      "description": "Filter documents to be checked with Grammarly.",
+                      "items": {
+                        "properties": {
+                          "language": {
+                            "description": "A language id, like `typescript`.",
+                            "type": "string"
+                          },
+                          "pattern": {
+                            "description": "A glob pattern, like `*.{md,txt}`.",
+                            "type": "string"
+                          },
+                          "scheme": {
+                            "description": "A Uri scheme, like `file` or `untitled`.",
```

Comment:
`markdownDescription` will look better for those.

Reply:
same as before -- setting is removed for now.